cmd/crddiff/main.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2022 Upbound Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"log"
+	"os"
+	"syscall"
+
+	"gopkg.in/alecthomas/kingpin.v2"
+
+	"github.com/upbound/uptest/crdschema"
+)
+
+func main() {
+	var (
+		app             = kingpin.New("crddiff", "A tool for checking breaking API changes between two CRD OpenAPI v3 schemas").DefaultEnvars()
+		baseCRDPath     = app.Arg("base", "The manifest file path of the CRD to be used as the base").Required().ExistingFile()
+		revisionCRDPath = app.Arg("revision", "The manifest file path of the CRD to be used as a revision to the base").Required().ExistingFile()
+	)
+	kingpin.MustParse(app.Parse(os.Args[1:]))
```

Comment:
I preferred a different command because I think they are conceptually different tasks, one is e2e testing using kuttl and the other is detecting & reporting breaking API changes. Instead of producing a single binary capable of doing conceptually different tasks, I preferred splitting the functionality into multiple binaries (which I believe is more inline with the Unix philosophy). But I don't have a strong opinion, we can also merge them into a single binary.

Reply:
The main advantage of merging into a single binary is packaging and distribution, i.e., we won't have to make separate releases and introduce another tool into the build submodule. 
I believe they could be combined conceptually as _tooling to test xp resources_ :)
So, I would vote for combining them due to practical reasons.

Reply:
Done. We now have `crddiff revision` and `crddiff self` as subcommands to a single uptest binary.